hvac/vitobloc/manifest.yml

```diff
@@ -0,0 +1,251 @@
+blueprint_spec: device/1.0
+
+display_name: Viessman Vitobloc 200
+description: Combined heat and power unit for natural gas and LPG operation
+icon: enapter-power-supply-unit
+author: enapter
+contributors:
+  - anataty
+  - Rina-an
+support:
+  email: support@enapter.com
+license: MIT
+verification_level: ready_for_testing
+
+communication_module:
+  product: ENP-VIRTUAL
+  lua_file: firmware.lua
+  dependencies: enapter-ucm ~> 0.1.8
+
+properties:
+  vendor:
+    display_name: Vendor
+    type: string
+  ADDRESS_CONFIG:
```

Comment:
compare with [this blueprint](https://github.com/Enapter/marketplace/tree/main/solar_inverters/sma_sunny_boy)

Reply:
compare what exactly? The name should be the same as in lua file, shouldn't it?